packages/terra-date-time-picker/tests/jest/DateTimePicker.test.jsx

```diff
@@ -116,3 +118,19 @@ it('should render a disabled date time picker', () => {
     </IntlProvider>));
   expect(datePicker).toMatchSnapshot();
 });
+
+it('Should not throw any errors while date value is outside of the Min, Max range and new time value is being entered ', () => {
+  const allMessages = Object.assign(messages, dateInputMessages, timeInputMessages);
+  const datePicker = mount((
```

Comment:
Can you put more new lines around some of these blocks. It feels very cluttered here.

Reply:
Addressed